RecoLocalTracker/SiStripRecHitConverter/src/StripCPEfromTrackAngle.cc

```diff
@@ -25,29 +48,34 @@ float StripCPEfromTrackAngle::legacyStripErrorSquared(const unsigned N, const fl
   }
 }
 
-StripClusterParameterEstimator::LocalValues StripCPEfromTrackAngle::
-localParameters( const SiStripCluster& cluster, const GeomDetUnit& det, const LocalTrajectoryParameters& ltp) const {
+StripClusterParameterEstimator::LocalValues 
+StripCPEfromTrackAngle::localParameters( const SiStripCluster& cluster, const GeomDetUnit& det, const LocalTrajectoryParameters& ltp) const {
   
   StripCPE::Param const & p = param(det);
   SiStripDetId ssdid = SiStripDetId( det.geographicalId() );  
  
   LocalVector track = ltp.momentum();
-  track *= 
-    (track.z()<0) ?  std::abs(p.thickness/track.z()) : 
-    (track.z()>0) ? -std::abs(p.thickness/track.z()) :  
-                         p.maxLength/track.mag() ;
+  track *= -p.thickness/track.z();
```

Comment:
is this penalty bigger than a possible crash in downstream code and actual loss of data because of that?
a check for z=0 or isFinite can be added (we don't have to put mag() back)